ui/app/mainui/activitycenter/views/ActivityNotificationMessage.qml

```diff
@@ -2,57 +2,112 @@ import QtQuick 2.14
+    property int maximumLineCount: 2
 
-    bodyComponent: MessageView {
-        rootStore: root.store
-        messageStore: root.store.messageStore
-        messageContextMenu: root.messageContextMenu
-
-        messageId: notification.id
-        senderDisplayName: notification.message.senderDisplayName
+    property StatusMessageDetails messageDetails: StatusMessageDetails {
         messageText: notification.message.messageText
-        senderId: notification.message.senderId
-        senderOptionalName: notification.message.senderOptionalName
-        senderIcon: notification.message.senderIcon
         amISender: notification.message.amISender
-        messageImage: notification.message.messageImage
-        messageTimestamp: notification.timestamp
-        messageOutgoingStatus: notification.message.outgoingStatus
-        messageContentType: notification.message.contentType
-        senderTrustStatus: notification.message.senderTrustStatus
-        activityCenterMessage: true
-        activityCenterMessageRead: false
-        onImageClicked: Global.openImagePopup(image, root.messageContextMenu)
-        messageClickHandler: (sender, 
-                              point,
-                              isProfileClick,
-                              isSticker = false,
-                              isImage = false,
-                              image = null,
-                              isEmoji = false,
-                              ideEmojiPicker = false,
-                              isReply = false,
-                              isRightClickOnImage = false,
-                              imageSource = "") => {
-            if (isProfileClick) {
-                return Global.openProfilePopup(notification.message.senderId)
+        sender.id: notification.message.senderId
+        sender.displayName: notification.message.senderDisplayName
+        sender.secondaryName: notification.message.senderOptionalName
+        sender.trustIndicator: notification.message.senderTrustStatus
+        sender.profileImage {
+            width: 40
+            height: 40
+            name: notification.message.senderIcon || ""
+            assetSettings.isImage: notification.message.senderIcon.startsWith("data")
+            pubkey: notification.message.senderId
+            colorId: Utils.colorIdForPubkey(notification.message.senderId)
+            colorHash: Utils.getColorHashAsJson(notification.message.senderId, false, true)
+            showRing: true
+        }
+    }
+
+    property Component messageSubheaderComponent: null
+    property Component messageBadgeComponent: null
+
+    function openProfilePopup() {
+        activityCenterClose()
+        Global.openProfilePopup(notification.message.senderId)
+    }
+
+    bodyComponent: RowLayout {
+        id: messageRow
+        spacing: 8
+
+        Item {
+            implicitWidth: root.messageDetails.sender.profileImage.assetSettings.width
+            implicitHeight: profileImage.height
```

Comment:
Just use Layout.preferredWidth/Height; setting implicitWidth/Height inside a Layout won't work (if it does it's just by chance and you'll get a warning about it)

Reply:
fixed